cdap-security/src/test/java/io/cdap/cdap/security/authorization/AccessControllerInstantiatorTest.java

```diff
@@ -596,5 +598,10 @@ public AuthorizedResult<Set<GrantedPermission>> listGrants(Principal caller, Pri
       throws AccessException {
       return null;
     }
+
+    @Override
+    public PublishStatus publish(Queue<AuditLogContext> auditLogContexts) {
```

Comment:
This class is not only about audit logs, actually audit logs is not it's main goal. We should at least give a better name to the method.